### PR TITLE
Steppy

### PR DESCRIPTION
**Pull request by rye-rice**
## About the Pull Request
How dare you call this tegustation and not include the reptiles?
https://github.com/vlggms/tegustation/pull/18
## Why It's Good For The Game

image
image
image

## Did you test it?

who the fuck needs to test for steppy

## Changelog

:cl:
bugfix - Steppy works again.
/:cl:

### CODE REVIEW

**Comment by Samuuu3** [comment]
Steppy I've missed you

**Comment by rye-rice** [comment]
steppy my beloved

**Comment by Cupax3** [comment]
Squash please

File: code/modules/mob/living/simple_animal/hostile/retaliate/alligator.dm [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	icon_living = "gator_cool"
+	speed = 7
+	harm_intent_damage = 30
+	health = 150
+	maxHealth = 150
+/*
+/mob/living/simple_animal/hostile/retaliate/gator/steppy/cool/death()
+	. = ..()
+	var/turf/T = get_turf(src)
+	new /obj/item/clothing/glasses/hud/security/sunglasses(T)
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/hostile/retaliate/alligator.dm]: I suppose hud sunglasses have a different path, so you should search for it and place it here.

**Comment by Cupax3** [comment]
@retlaw34 Test your changes locally before, too. Thanks.

**Review by Cupax3 — CHANGES_REQUESTED** [review]
Test it locally & remove the list tag from the :cl: please, thank you.

EDIT: And squash too, thanks.

**Pull request closed without merging**